pallets/staking/src/lib.rs

```diff
@@ -334,6 +345,7 @@ use sp_std::{
     result,
 };
 
+const DUMMY_VALUE: u8 = 0u8;
```

Comment:
How and why the dummy value is used?

Reply:
for:
```/// Clears both snapshots of stakers.
    fn kill_stakers_snapshot() {
        <SnapshotValidators<T>>::kill();
        <SnapshotNominators<T>>::kill();

        // Remove Stash's liquidity valuation details
        <StashStakedValuation<T>>::remove_prefix(DUMMY_VALUE);
    }```
    If we don't use DUMMY_VALUE, then we can't use remove_prefix, then we have to remove each item individually.

Reply:
Ok, will take a deeper look into it